### PR TITLE
Add Deployed ML models dashboard blog

### LINKED ISSUE #1409: [Blog] ML Commons Admin UI
**Describe the blog post your would like to write**
The ML-Commons admin UI is an intuitive and powerful interface for monitoring the responsing status of your machine learning models across various ML nodes. We are going to write this blog to discuss features and technical challenges.

**What is the title of the blog post?**
ML Commons Admin UI

**Who are the authors?**
wonglam, yych

**What is the proposed posting date?**
23th, March

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Fixes #1409 

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2023-04-05-ml-deployment-status.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  "See machine learning model status in OpenSearch Dashboards"
+authors:
+- naarcha
+date: 2023-04-05
+categories:
+ - technical-post
+meta_keywords: machine learning, ml commons, opensearch dashboards, model-serving framework, deployed models dashboard
+meta_description: Learn how to check your deployed machine learning models deployment status inside OpenSearch Dashboards.
```
Comment (pajuric) [inline _posts/2023-04-05-ml-deployment-status.md:10]: Please update the meta with the following changes: 
meta_keywords:  machine learning, ML Commons for OpenSearch, deployed models dashboard 
meta_description: Test the experimental deployed model dashboard functionality of OpenSearch Dashboards to view the responsiveness of ML nodes in your cluster.

**Review by wanglam — DISMISSED** [review]

File: _posts/2023-04-05-ml-deployment-status.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+excerpt: Before OpenSearch 2.6, checking the status of a deployed machine learning (ML) model in OpenSearch could only be done by using the ML Commons Profile API. Although the Profile APi returned data organized by ML nodes, it could not tell you the status of a specific model deployed on that node. 
+---
+
+Before OpenSearch 2.6, checking the status of a deployed machine learning (ML) model in OpenSearch could only be done by using the [ML Commons Profile API](https://opensearch.org/docs/latest/ml-commons-plugin/api/#profile). Although the Profile APi returned data organized by ML nodes, it could not tell you the status of a specific model deployed on that node.
```
Comment (wanglam) [inline _posts/2023-04-05-ml-deployment-status.md:14]: @Naarcha-AWS  As @pajuric said could we link to the opensearch 2.6 introduction blog? Here is the article link: https://opensearch.org/blog/introducing-opensearch-2-6/

Comment (Naarcha-AWS) [reply _posts/2023-04-05-ml-deployment-status.md:14]: Can do!

Comment (Naarcha-AWS) [reply _posts/2023-04-05-ml-deployment-status.md]: Done.

**Review by wanglam — DISMISSED** [review]

File: _posts/2023-04-05-ml-deployment-status.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+If you deploy a large number of models, you might it hard to locate to model you need with the Profile API. Luckily, you can search and filter through all deployed models using:
+
+- Model name
+- Model ID
+- Model status
+
+Furthermore, the deployed model dashboard can automatically refresh data according to a selected time cycle. To set the refresh cycle, select the **Clock** icon, set the refresh time, and then select **Start**.
+
+<img src="/assets/media/blog-images/2023-04-05-ml-deployment-status/ml-model-search.gif" alt="Search and filter through deployed models"/>{: .img-fluid }
```
Comment (natebower) [inline _posts/2023-04-05-ml-deployment-status.md:45]: The image needs to be introduced in the text that precedes it.

**Review by natebower — CHANGES_REQUESTED** [review]
@Naarcha-AWS Please see my comments and changes and tag me when you are finished addressing them so that I can approve and push forward. Thanks!

File: _posts/2023-04-05-ml-deployment-status.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+Before [OpenSearch 2.6](https://opensearch.org/blog/introducing-opensearch-2-6/), checking the status of a deployed machine learning (ML) model in OpenSearch could only be done by using the [ML Commons Profile API](https://opensearch.org/docs/latest/ml-commons-plugin/api/#profile). Although the Profile API returned data organized by ML nodes, the response could not tell you the status of a specific model deployed on an ML node. 
+
+The ML team is happy to announce the first piece of ML functionality in OpenSearch Dashboards, the [deployed models dashboard](https://opensearch.org/docs/latest/ml-commons-plugin/ml-dashboard/#deployed-models-dashboard), as an experimental feature. The deployed models dashboard gives you a view into the responsiveness of each deployed model on each ML node in your cluster.
+
+## Viewing deployed ML status at the model level
+
+You can find the the deployed model dashboard inside the **Machine Learning** section of OpenSearch Dashboards, as shown in the following image.
```
Comment (natebower) [inline _posts/2023-04-05-ml-deployment-status.md:22]:
```suggestion
You can find the the deployed model dashboard in the **Machine Learning** section of OpenSearch Dashboards, as shown in the following image.
```

File: _posts/2023-04-05-ml-deployment-status.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+The detailed view prevents you from having to manually call the Profile API and parse through complex return results to see which ML nodes are responsive. Instead, the detailed view tells you which ML nodes, by Node ID, are responding or not responding, which is very useful when troubleshooting deployment issues.
+
+
+## Simplifying model filtering and searching
+
+If you deploy a large number of models, you might find it difficult to locate the model you need with the Profile API. Luckily, you can search and filter through all deployed models by using:
+
+- The model name.
+- the model ID
```
Comment (natebower) [inline _posts/2023-04-05-ml-deployment-status.md:48]:
```suggestion
- The model ID.
```

**Review by natebower — DISMISSED** [review]
Thanks @Naarcha-AWS! @pajuric This is ready to publish.

**Review by wanglam — DISMISSED** [review]

**Comment by pajuric** [comment]
@kolchfa-aws @Naarcha-AWS - Has this been finalized or are you waiting on editorial approval?

**Comment by natebower** [comment]
> @kolchfa-aws @Naarcha-AWS - Has this been finalized or are you waiting on editorial approval?

https://github.com/opensearch-project/project-website/pull/1496#pullrequestreview-1373297804

**Comment by kolchfa-aws** [comment]
@Naarcha-AWS I noticed you have one extraneous commit in this PR ([8641bf1](https://github.com/opensearch-project/project-website/pull/1496/commits/8641bf198b3cdfefb5dfbbef472bb2312e8204fc)).

File: _posts/2023-04-05-ml-deployment-status.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title:  "You can now see ML model status in OpenSearch Dashboards"
+authors:
+- wonglam
+- yych
+- naarcha
+date: 2023-04-05
```
Comment (pajuric) [inline _posts/2023-04-05-ml-deployment-status.md:8]: Publishing date will need to be updated. Please let me know the new scheduled date.

Comment (natebower) [reply _posts/2023-04-05-ml-deployment-status.md:8]: Publishing date is at the discretion of Marketing. Please update as appropriate.

**Review by dtaivpp — APPROVED** [review]
Content looks good. Going to see about getting DCO resolved

**Comment by dtaivpp** [comment]
Closing in light of #1523

**Pull request closed without merging**